wolfcrypt/src/port/nrf51.c

```diff
@@ -171,13 +165,16 @@ static void rtc_config(void)
     uint32_t err_code;
 
     // Start the internal LFCLK XTAL oscillator
-    err_code = nrf_drv_clock_init(NULL);
+    err_code = nrf_drv_clock_init();
```

Comment:
We need to maintain backwards compatibility for older nRF5 SDK's. This current code is based on nRF5 SDK v10. (https://github.com/dgarske/nRF51_SDK_10). Either check for an nRF5 SDK version macro or use the `WOLFSSL_NRF52` macro.

Reply:
The old API was:
`ret_code_t nrf_drv_rng_bytes_available(uint8_t * p_bytes_available);`
it is currently:
`void nrf_drv_rng_bytes_available(uint8_t * p_bytes_available);`
This API is present in both SDK v12.3 (newest SDK supporting the NRF51 series) and SDK v15.2 (newest SDK). The old API is still present in SDK v10. I should probably use a macro to choose which API to use but I'm not sure which one to use.